typings/index.d.ts

```diff
@@ -3251,6 +3259,21 @@ declare module 'discord.js' {
 
   type Status = number;
 
+  export class Sticker extends Base {
+    constructor(client: Client, data: object);
+    public asset: string;
+    public readonly createdTimestamp: number;
+    public readonly createdAt: Date;
+    public description: string;
+    public format: StickerFormatTypes;
+    public id: Snowflake;
+    public name: string;
+    public packID: Snowflake;
+    public previewAsset: string | null;
+    public tags: string[];
+    public url: string;
```

Comment:
This should be `readonly` now to reflect 567009de47a395959d8b25a5dbc6ce65ac8090da
```suggestion
    public readonly url: string;
```